example-gto/code/README.md

```diff
@@ -0,0 +1,33 @@
+# GTO Get Started
+
+This is an auto-generated repository for use in
+[GTO Get Started](https://github.com/iterative/gto/blob/main/README.md).
```

Comment:
can be a just link to the repo itself I think

Reply:
Done